File: Gym_.NET-master/Gym.API/Domain/Models/News.cs
using System;
using System.Numerics;
using System.Collections.Generic;

namespace Gym.API.Domain.Models
{

    public class News
    {
        public int IdNews { get; set; }
        public string Title { get; set; }

        public string Text { get; set; }
        public DateTime Date { get; set; }

    }
}

File: Gym/app/src/main/java/com/example/gym/models/News.kt
package com.example.gym.models

import com.google.gson.annotations.SerializedName
import java.math.BigInteger
import java.util.*

data class News (

    @SerializedName("idNews") val idNews : Int,
    @SerializedName("date") val date : String,
    @SerializedName("title") var title: String,
    @SerializedName("text") var text : String
)